Doug/Services/EventService.cs

```diff
@@ -23,6 +32,15 @@ public void MessageReceived(MessageEvent message)
             {
                 _coffeeBreakService.CountParrot(message.User, message.Channel, DateTime.UtcNow);
             }
+
+            List<string> Words = new List<string>() { "mcd0", "mcdo", "mc d0", "mc do", "mcdonald", "mc donald", "mcd0nalds", "mcd0nald$", "mcd0n4lds", "mcd0n4ld$", "mc d0n4ld$", "mc d0nald$" };
+            bool isMcdonald = message.User == GAB_ID && Words.Contains(message.Text.ToLower());
```

Comment:
should be `message.Text.contains("mcdo")`, since the real message would look like : "On va tu au mcdo?"